[Brother] Add information about default password (#30229)
home-assistant/home-assistant.github.io: source/_integrations/brother.markdown

## Configuring the printer

> To enable SNMP, navigate to the printer's web interface (for example: `http://192.168.5.6`) and turn it on under Network / Protocol / SNMP. To see this option, you'll need to log in using the password field at the top of the page. For some printers, the default password is printed on a sticker on the back of the printer, preceded by "Pwd:". If the printer does not have a password on the sticker, the default password is "initpass".

Reviewer: ```
To see this option, you'll need to log in using the password field at the top of the page.
...the default password is "initpass"
```
This is not true for all printer models. Some models do not have a password for the www panel by default.

> To enable SNMP, navigate to the printer's web interface (for example: `http://192.168.5.6`) and turn it on under Network / Protocol / SNMP. To see this option, you'll need to log in using the password field at the top of the page. For some printers, the default password is printed on a sticker on the back of the printer, preceded by "Pwd:". If the printer does not have a password on the sticker, the default password is "initpass".

Reviewer 2: ```suggestion
To enable SNMP, navigate to the printer's web interface (for example: `http://192.168.5.6`) and turn it on under Network / Protocol / SNMP. For some models, access to the web interface is password-protected. For some printers, the default password is printed on a sticker on the back of the printer, preceded by "Pwd:". If the printer does not have a password on the sticker, the default password is "initpass".
```
@bieniu, would this wording work?

Reviewer: Oh my, I completely forgot about this PR. Yes, now it sounds good.

Reviewer 2: No worries. Thank you for your feedback!
